packages/remotes/pyproject.toml

```diff
@@ -42,7 +42,7 @@ requires-python = ">=3.11"
 dev = [
     "ether0",
     "ether0.remotes[serve]",
-    "tensorboard>=2.19",  # Indirect dependency we pin to keep recent
+    "tensorboard>=2.18",  # Indirect dependency we pin to keep recent
```

Comment:
I loosened the pinning here to allow for package resolve, as `lighteval==0.10.0` requires `numpy` v1: https://github.com/huggingface/lighteval/issues/416